Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Set image from base on offical node lts alpine
+ARG VERSION=18-bullseye-slim
+FROM node:$VERSION
+
+# Set working directory
+WORKDIR /app
+
+# Copy all files
+COPY . .
+
+# Install dependencies
+RUN yarn install --frozen-lockfile
+
+# Build app
+RUN yarn build
+
+# Expose the listening port
+EXPOSE 3000
+
+# Run yarn start script when container starts
+CMD yarn start
```

Comment:
possible to make it multi-stage?
reference: [Next.js multi-stage Docker file](https://cj-hewett.medium.com/next-js-multi-stage-docker-file-f41454af53c1)